src/Mod/TechDraw/Gui/CommandExtensionDims.cpp

```diff
@@ -141,15 +141,15 @@ DEF_STD_CMD_A(CmdTechDrawExtensionInsertDiameter)
 CmdTechDrawExtensionInsertDiameter::CmdTechDrawExtensionInsertDiameter()
     : Command("TechDraw_ExtensionInsertDiameter")
 {
-    sAppModule = "TechDraw";
-    sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Insert '⌀' sign");
-    sToolTipText = QT_TR_NOOP("Insert '⌀' sign in a measure\n\
-    - select many measures\n\
-    - click this button");
-    sWhatsThis = "TechDraw_ExtensionInsertDiameter";
-    sStatusTip = sToolTipText;
-    sPixmap = "TechDraw_ExtensionInsertDiameter";
+    sAppModule      = "TechDraw";
+    sGroup          = QT_TR_NOOP("TechDraw");
+    sMenuText       = QT_TR_NOOP("Insert '⌀' Prefix");
+    sToolTipText    = QT_TR_NOOP("Insert a '⌀' symbol at the beginning of the dimension text:<br>\
+- Select one or more dimensions<br>\
+- Click this tool");
```

Comment:
A newline should be enough. No need to add `<br>` (also it confuses the translators)

Reply:
With the new tooltip HTML formatting `<br>` is required.
Forum topic:
https://forum.freecadweb.org/viewtopic.php?t=62587
Image:
![tooltip-no-linebreaks](https://user-images.githubusercontent.com/70520633/150695190-febf8606-3651-4011-8575-1e20469cb439.png)

Reply:
Are you kidding?!? :facepalm: that means _so_ many translations will need to be redone!

Reply:
I do not know how many tooltips contain linebreaks, but it is not that common.